packages/blueprints/src/base/showstyle/helpers/sources.ts

```diff
@@ -24,14 +29,18 @@ export function findSource(input: string | number | boolean | undefined, type: S
 }
 
 export function getSourceInfoFromRaw(config: StudioConfig, rawInfo: RawSourceInfo): SourceInfo {
-	let sourcesOfType = Object.values<InputConfig>(config.atemSources).filter((s) => s.type === rawInfo.type)
+	let input: InputConfig | undefined = undefined
```

Comment:
instead of the casting below, what about making this:
```suggestion
	let input: InputConfig | VmixInputConfig | undefined = undefined
```
That might keep typescript happy without the casting